### ISSUE TITLE
Smart playlist of subsonic empty

### ISSUE THREAD

**Issue by ouafnico** [clementine-player/Clementine#5698]
Hi
I'm using clementine connected to a subsonic.
It works well except the smart playlist, they are all empty. They seems to do not get playlists from subsonic server.

### System information

- Operating System: Debian 9
- Clementine version: 1.3.1+git276-g3485bbe43+dfsg-1

Thanks

**Comment by llucps** [comment]
Yes, it happens to me too in MacOS High Sierra and latest clementine build 1.3.1-477-ga65dabc, although this feature has never worked on any version whatsoever.

Anyone can shed some light here? Has it ever worked before?

Thanks,

**Comment by dbogdanov** [comment]
Same problem. All smart playlists are empty except "Random Songs" which works.

System information: Clementine 1.4 rc2 installed from APT on Ubuntu 21.04.
